.github/workflows/mirror_repos.yml

```diff
@@ -72,9 +72,29 @@ jobs:
           git config --global user.name AztecBot
           git config --global user.email tech@aztecprotocol.com
 
+          monorepo_url="$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
+          # list all aztec-packages tags, take the "highest" version
+          monorepo_tag="$(git tag --list aztec-packages-v* | sort --version-sort | tail -1)"
+          monorepo_protocol_circuits_path="yarn-project/noir-protocol-circuits"
+          nargo_file="$SUBREPO_PATH/aztec/Nargo.toml"
+
+          # match lines like this:
+          # protocol_types = { path = "../../noir-protocol-circuits/src/crates/types" }
+          # and replace with
+          # protocol_types = { git="https://github.com/aztecprotocol/aztec-packages", tag="aztec-packages-v0.16.9", directory="yarn-project/noir-protocol-circuits/src/crates/types" }
+          sed --regexp-extended --in-place \
+            "s;path\s*=\s*\".*noir-protocol-circuits(.*)\";git=\"$monorepo_url\", tag=\"$monorepo_tag\", directory=\"$monorepo_protocol_circuits_path\1\";" \
+            $nargo_file
+
+          git commit --all --message "chore: replace relative paths to noir-protocol-circuits"
```

Comment:
do we allow `chore:` in commit names?

Reply:
I think I have used it before. But not sure it that just my breaking conventions 😬

Reply:
yep that should be allowed